ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -95,6 +95,10 @@ tests:
     - as: test
       cli: latest
       commands: ./test-prow-e2e.sh ceph
+      credentials:
+      - mount_path: /var/run/operator-secret
+        name: operator-secret
+        namespace: test-credentials
```

Comment:
I believe you'll need these changes in openshift-console-release-4.8.yaml and openshift-console-release-4.9.yaml

Reply:
Thanks for pointing that.